spec/factories/omniauth.rb

```diff
@@ -0,0 +1,33 @@
+FactoryGirl.define do
+
+  factory :auth_hash, class: OmniAuth::AuthHash do
+    skip_create
+
+    initialize_with do
+      OmniAuth::AuthHash.new({
+        provider: provider,
+        uid: uid,
+        info: {
+          email: email
+        }
+      })
```

Comment:
I understand that, but that is why I added traits. My intent was to always use a trait in my tests.